src/utils/chain.js

```diff
@@ -37,7 +37,9 @@ export const supportedChains = {
     hub_sort_order: 6,
     providers: ['walletconnect'],
     // , 'portis', 'fortmatic'
-    rpc_url: `https://${process.env.REACT_APP_RPC_URI}.rinkeby.rpc.rivet.cloud/`,
+    rpc_url:
```

Comment:
Would like to know why.

Reply:
my mistake. This change will be rollbacked in the next PR update